diagnostic.md

```diff
@@ -18,33 +18,32 @@ In what file does the above code go?
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+db:migrate checks which missing migrations still need to be applied to the database without caring about the previouse ones.
```

Comment:
`bundle exec rake db:migrate` executes all of the database migration files in `db/migrate` and generates a new database schema file, `db/schema.rb`.